### PR TITLE
Add issue template and improve pull request template

### PR DESCRIPTION
**Pull request by divyam3897**
Added an issue and pull request template in .github/ following https://github.com/blog/2111-issue-and-pull-request-templates

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+**Expected behavior:**
+<!-- Describe what the behavior would be without the bug. -->
+
+**Steps to reproduce:**
+<!-- If you are able to illustrate the bug or feature request with an example, please provide steps to reproduce -->
+
+**Related code:**
+
+```
```
Comment (lex111) [inline .github/ISSUE_TEMPLATE.md]: Code block should not be wrapped in a comment?

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+insert any relevant code here else remove this section
+```
+
+**Other information:**
+<!-- List any other information that is relevant to your issue. Stack traces, related issues, suggestions on how to fix, Stack Overflow links, forum links, etc. -->
+
+**Would you like to work on the issue?**
+
+Please let us know if you can work on it or the issue should be assigned to someone else.
```
Comment (lex111) [inline .github/ISSUE_TEMPLATE.md]: Maybe this should be wrapped in a HTML comment?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+Fixes issue #[Add issue number here. If you do not solve the issue entirely, please change the message e.g. "First steps for issues #IssueNumber]
```
Comment (lex111) [inline .github/PULL_REQUEST_TEMPLATE.md]: @jenkoian if there is no issue, does it need to be deleted?

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (divyam3897) [reply .github/ISSUE_TEMPLATE.md]: Why in a comment?

File: .github/ISSUE_TEMPLATE.md (hunk 2 above)
Comment (divyam3897) [reply .github/ISSUE_TEMPLATE.md]: do you mean to not display it in the template also?

**Comment by jenkoian** [comment]
Thanks @divyam3897 I like the changes to the PR template. The issue template I think would put me off submitting an issue and would worry it would do the same for others. I'm not against a template, but would prefer something much more lightweight. Are you able to change to something more inline with the PR template?

**Comment by divyam3897** [comment]
@jenkoian I understand :D Though it was the standard one that I have used with my projects/org. I can reduce it's difficulty if you want?

**Comment by divyam3897** [comment]
@jenkoian done 😄

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (jenkoian) [reply .github/PULL_REQUEST_TEMPLATE.md]: Yeah or left blank or something

**Comment by jenkoian** [comment]
Thanks @divyam3897 !

**Pull request merged**